frontend/pulse3d/src/components/account/PasswordForm.js

```diff
@@ -60,7 +60,8 @@ export default function PasswordForm({ onChangePassword, setErrorMsg, password1,
     // this removes all borders/error messages once a user has backspaced to an empty input field
     if (password1.length > 0) {
       // min 10 chars, one number, one uppercase, one lowercase, one special character
-      const reqRegex = /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[@$!%*?&]).{10,}/;
+      const reqRegex =
+        /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#$%^&*()~`|<>,.+=_"':;?/\\\[\]\{\}-]).{10,}/;
```

Comment:
I meant to add a comment to the asana ticket, but could you also add the allowed special characters to the tooltip text on line 101